autofs-5.1.4 - fix program usage message

From: Ian Kent <raven@themaw.net>

There is a discrepancy between the list of options in the usage
message of automount(8) and the options listed in the man page.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG          |    1 +
 daemon/automount.c |    6 +++++-
 2 files changed, 6 insertions(+), 1 deletion(-)

--- autofs-5.1.4.orig/CHANGELOG
+++ autofs-5.1.4/CHANGELOG
@@ -27,6 +27,7 @@ xx/xx/2018 autofs-5.1.5
 - fix fd leak in rpc_do_create_client().
 - add-man page note about extra slashes in paths.
 - covarity fixes.
+- fix program usage message.
 
 19/12/2017 autofs-5.1.4
 - fix spec file url.
--- autofs-5.1.4.orig/daemon/automount.c
+++ autofs-5.1.4/daemon/automount.c
@@ -1893,9 +1893,13 @@ static void usage(void)
 		"	-h --help	this text\n"
 		"	-p --pid-file f write process id to file f\n"
 		"	-t --timeout n	auto-unmount in n seconds (0-disable)\n"
+		"	-M --master-wait n\n"
+	        "			maximum wait time (seconds) for master\n"
+	        "			map to become available\n"
 		"	-v --verbose	be verbose\n"
 		"	-d --debug	log debuging info\n"
-		"	-D --define	define global macro variable\n"
+		"	-Dvariable=value, --define variable=value\n"
+		"			define global macro variable\n"
 		"	-f --foreground do not fork into background\n"
 		"	-r --random-multimount-selection\n"
 		"			use ramdom replicated server selection\n"
